ark/analysis/spatial_analysis.py

```diff
@@ -58,12 +59,13 @@ def calculate_channel_spatial_enrichment(dist_matrices_dict, marker_thresholds,
                              "eccentricity", "major_axis_length", "minor_axis_length",
                              "perimeter", "fov"]
 
-    # Error Checking
-    if not np.isin(excluded_colnames, all_data.columns).all():
-        raise ValueError("Column names were not found in Expression Matrix")
+    # check if included fovs found in fov_col
+    misc_utils.verify_in_list(included_fovs=included_fovs,
+                              all_data_fov_col_values=all_data[fov_col].unique())
```

Comment:
I think we could change the variable name be more interpretable, something like `fov_names`, `unique_fovs`, etc